src/data/staking-products.json

```diff
@@ -636,6 +636,35 @@
         "eventAction": "Clicked",
         "eventName": "Clicked Sensei Node go to link"
       }
+    },
+    {
+      "name": "Figment",
+      "imageName": "Figment",
+      "hue": 52,
+      "launchDate": "2021-12-31",
+      "url": "https://figment.io/",
+      "audits": [],
+      "minEth": 32,
+      "monthlyFee": 30,
+      "monthlyFeeUnit": "% execution rewards",
+      "isFoss": false,
+      "hasBugBounty": false,
+      "isTrustless": false,
+      "isPermissionless": false,
+      "pctMajorityExecutionClient": 85,
+      "pctMajorityConsensusClient": 0,
+      "isSelfCustody": false,
```

Comment:
@ivanszeftel As noted, this is from it being private
<img width="600" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/b716375f-dfa6-427f-8288-68b6b85b1472">
> For safety reasons, the access to the validator private keys (signing keys) is strictly limited to the service.